Source/Scene/Cesium3DTileset.js

```diff
@@ -892,13 +879,8 @@ define([
 
     function raiseLoadProgressEvents(tileset, frameState) {
         var eventsToRaise = tileset._loadProgressEventsToRaise;
-        var length = eventsToRaise.length;
-        for (var i = 0; i < length; ++i) {
-            var numberOfPendingRequests = eventsToRaise[i].numberOfPendingRequests;
-            var numberProcessing = eventsToRaise[i].numberProcessing;
-
-            frameState.afterRender.push(evenMoreComplicated(tileset, numberOfPendingRequests, numberProcessing));
-        }
+        frameState.afterRender.push(evenMoreComplicated(tileset, tileset._statistics.numberOfPendingRequests,
```

Comment:
Looks like this is the only place that `evenMoreComplicated` is called. Should I remove it?